rule/templates/_rule.js

```diff
@@ -8,6 +8,9 @@
 // Rule Definition
 //------------------------------------------------------------------------------
 
+/**
+ * @type {import('eslint').Rule.RuleModule}
```

Comment:
Do we need to include `@types/eslint` to make this work?

Reply:
My understanding is no. VSCode has [Automatic Type Acquistion](https://code.visualstudio.com/docs/nodejs/working-with-javascript#_typings-and-automatic-type-acquisition) where it automatically retrieves the corresponding `@types/` package for your dependencies.

Reply:
Huh, that’s cool. I wonder if that has any negative effect when running `tsc`? I don’t want to please VS Code only to anger TypeScript. Can you double check that?

Reply:
These rules are JS files so I don't think anyone will be running `tsc` with them?

However, I checked what happens when using `@type` in a TS file, and TS shows a non-blocking hint:

```
JSDoc types may be moved to TypeScript types. ts(80004)
```

Reply:
Fair enough. I run `TSC` on JavaScript files frequently to generate .d.ts files from JSDoc comments. :)